src/app/components/BookList/index.tsx

```diff
@@ -16,11 +16,12 @@ interface Book {
 
 interface Props {
   books: Book[];
+  onPress: (event: GestureResponderEvent) => void;
```

Comment:
I think the `event: GestureResponderEvent` is not necessary => `onPress: () => void;`